sql_generators/glean_usage/event_monitoring_live.py

```diff
@@ -212,7 +209,7 @@ def generate_across_apps(
                     event_tables_per_dataset[dataset] = events_table_overwrites[
                         app_name
                     ]
-                else:
+                elif app_name not in skipped_apps:
```

Comment:
CI failure is because the app is still in the view that unions all the apps so the app needs to be removed from the list or `render_kwargs` needs to be changed

Reply:
Actually the error is for `accounts_backend` so something else is happening here but I don't know what

Reply:
mh, the stage deploy pipeline doesn't seem to be able to deploy the materialized views

Reply:
Oh right, I was talking to Arkadiusz about this as well.  I think it's because initialize only runs for queries with `is_init()` https://github.com/mozilla/bigquery-etl/blob/9fc513ba9f10286accca16ebe69e2cf18c64b33c/bigquery_etl/cli/query.py#L1490